src/main/java/com/suse/salt/netapi/calls/modules/Locate.java

```diff
@@ -0,0 +1,182 @@
+package com.suse.salt.netapi.calls.modules;
+
+import com.google.gson.annotations.SerializedName;
+import com.google.gson.reflect.TypeToken;
+import com.suse.salt.netapi.calls.LocalCall;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * salt.modules.locate
+ * Module for using the mlocate utilities.
+ */
+public class Locate {
+
+    private static final LocalCall<List<String>> VERSION = new LocalCall<>(
+            "locate.version",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<List<String>>() {
+            });
+
+    private static final LocalCall<List<String>> UPDATEDB = new LocalCall<>(
+            "locate.updatedb",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<List<String>>() {
+            });
+
+    private static final LocalCall<Stats> STATS = new LocalCall<>("locate.stats",
+            Optional.empty(), Optional.empty(),
+            new TypeToken<Stats>() {
+            });
+
+    /**
+     * Locate module result object
+     */
+    public static class Stats {
+        private Long files;
```

Comment:
We generally use primitive values whenever possible. I think the `Result` class may have slipped through code review. It would be good if you could change to primitive values where possible.